lib/ansible/parsing/vault/__init__.py

```diff
@@ -60,14 +61,33 @@
 display = Display()
 
 
-b_HEADER = b'$ANSIBLE_VAULT'
```

Comment:
Looks like `ansible.parsing.dataloader` still imports this: https://dev.azure.com/ansible/ansible/_build/results?buildId=114574&view=logs&j=62bae1aa-811d-5f3b-a44f-cf68dc674f86&t=01808b31-9561-58ed-cfa7-cc2d70afc012&l=3995